mp/src/game/server/neo/neo_bloom_controller.cpp

```diff
@@ -0,0 +1,69 @@
+//========= Copyright Valve Corporation, All rights reserved. ============//
+//
+// Purpose: 
+//
+//=============================================================================
+
+#include "cbase.h"
+#include "baseentity.h"
+#include "entityoutput.h"
+#include "convar.h"
+// memdbgon must be the last include file in a .cpp file!!!
+#include "tier0/memdbgon.h"
+
+//-----------------------------------------------------------------------------
+// Purpose: Entity that controls player's tonemap
+//-----------------------------------------------------------------------------
+class CNeoBloomController : public CPointEntity
+{
+	DECLARE_CLASS( CNeoBloomController, CPointEntity );
+public:
+	DECLARE_DATADESC();
+	DECLARE_SERVERCLASS();
+
+	void	Spawn( void );
+	int		UpdateTransmitState( void );
+
+	// Inputs
+	void	InputSetBloomScale( inputdata_t &inputdata );
+
+private:
+	CNetworkVar( float, m_flBloomAmountSetting);
+};
+
+LINK_ENTITY_TO_CLASS( neo_bloom_controller, CNeoBloomController );
+
+BEGIN_DATADESC( CNeoBloomController )
+	DEFINE_FIELD( m_flBloomAmountSetting, FIELD_FLOAT ),
+	DEFINE_INPUT( m_flBloomAmountSetting, FIELD_FLOAT, "bloomamount"),
```

Comment:
```suggestion
	DEFINE_KEYFIELD( m_flBloomAmountSetting, FIELD_FLOAT, "bloomamount"),
	DEFINE_INPUTFUNC(FIELD_FLOAT, "SetBloomScale", InputSetBloomScale),
```

On my side, this entity doesn't work at all as you seem to have defined your keyvalue and input incorrectly. This should fix it?